desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -1899,6 +1899,7 @@
   [[abfs_clusters]]
     # Default ABFS cluster
     [[[default]]]
+      ## enable_defaultfs_from_coresite=true
```

Comment:
Same, lets not add this flag here to let config be clean.

Reply:
If a user wants to set the flag and if the flag does not exist in the ini file then the user will not know that there is an option available to set.